core-feature-pack/src/main/resources/content/bin/jboss-cli.ps1

```diff
@@ -1,6 +1,6 @@
-### -*- Power Shell file -*- ################################################
+#############################################################################
 #                                                                          ##
-#    JBoss CLI Script for interacting with the standalone server           ##
+#    WildFly CLI Script for interacting with the server                    ##
```

Comment:
This kind of thing just makes maintaining EAP a PITA, as we are certain to get complaints leading to silly code differences between branches. Let's just get rid of the name altogether in these. If people are confused about what software the script is for, they probably shouldn't touch them anyway. :)